src/main/java/net/sf/jabref/JabRefGUI.java

```diff
@@ -192,6 +196,19 @@ private void openWindow() {
         if (!bibDatabases.isEmpty()) {
             JabRefGUI.getMainFrame().getCurrentBasePanel().getMainTable().requestFocus();
         }
+
+        boolean isSharedDatabaseEdited = Globals.prefs.getBoolean(JabRefPreferences.SHARED_DATABASE_LAST_EDITED);
+        if (isSharedDatabaseEdited) {
+            boolean isFocused = Globals.prefs.getBoolean(JabRefPreferences.SHARED_DATABASE_LAST_FOCUSED);
+            String keywordSeparator = Globals.prefs.get(JabRefPreferences.KEYWORD_SEPARATOR);
+
+            try {
+                new SharedDatabaseUIManager(mainFrame, keywordSeparator).openLastSharedDatabaseTab(isFocused);
+            } catch (ClassNotFoundException | SQLException | DatabaseNotSupportedException e) {
+                LOGGER.error("Failed to restore shared database. Use connection dialog to connect.");
```

Comment:
Please add the `exception e` as second parameter to the logger.error (there is already an overload for that=